src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
-                    <li> - bringing it away from such an examination;</li>
-                    <li> - delivering it, by previous arrangement, to a place where relevant work is to be done on it following
-                        receipt of a refusal to issue an Individual Approval Certificate;</li>
-                    <li> - bringing it away from a place where relevant work has been done on it.</li>
+        <div class="section-title-border" id="defect-details-subtitle-border">
+            <h1> Required Standard(s) Not Met: </h1>
+        </div>
+        <div class="section-text-wrapper">
+            <!-- List incoming defects here -->
+        </div>
+    </div>
+
+    <p> I hereby refuse an Individual Approval Certificate in respect of the above described vehicle on the
+        grounds that the vehicle fails to comply with the relevant requirements prescribed under The Road Vehicles
+        (Approval) Regulations 2020 </p>
+
+    <!-- Date/name table here -->
+    <table class="inspector-detail-table">
+        <tr>
+            <td></td>
+            <td class="table-row-cell"> <b>Reapplication Required By: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell"> <b>Signed: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"> <b>Date: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell"> <b>Name: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"> <b>Station: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell" colspan="2"> <p><b>The Following additional defects may result in the vehicle not complying with the requirements
+                of the Road Vehicles (Construction and Use) Regulations 1986. (See notes overleaf) </b></p>
+                <!-- Put AdditionalDefects here -->
+            </td>
+        </tr>
+    </table>
+
+
+    <!-- Data Protection Section -->
+    <div class="section-border" id = "data-prot-section">
+        <div class="section-title-border">
+            <h1> DATA PROTECTION STATEMENT 2019 </h1>
+        </div>
+        <div class="section-text-wrapper">
+            <p> <b>DATA PROTECTION</b> - We collect and store your personal data so that we can provide you with a
+                notification of refusal to issue an Individual Approval Certificate (IAC). </p>
+            <p> We may share your personal data if we have a lawful reason. For example, as part of a criminal
+                investigation or to prevent fraud. Find out more at <a href="www.gov.uk/dvsa/privacy">www.gov.uk/dvsa/privacy</a></p>
+        </div>
+    </div>
+
+    <!-- footer -->
+    <div class="footer">
+        <p class="footer-left"> <b class="bold-title">IVA30VTA (DVSA0842) </b> </p>
+        <p class="footer-mid"> <b> 1 of 2 </b> </p>
+        <p class="footer-right"> <b>Version 1.0 Oct 2023</b> </p>
```

Comment:
Another one potentially to check with PO, should this be updated to now reflect the recent change in Feb?

Should it now be whatever date this is merged into develop?

Reply:
@m-coslett